.github/workflows/release.yml

```diff
@@ -3,18 +3,36 @@ on:
     tags:
     - 'v*'
 
-name: publish release
+name: Release
 
 jobs:
-  build:
-    name: Create new Github release
+  linter:
+    name: "Run Golang linter"
+    uses: kubewarden/kubewarden-controller/.github/workflows/golangci-lint.yml@main
+  unit-tests:
+    name: "Run unit tests"
+    uses: kubewarden/kubewarden-controller/.github/workflows/unit-test.yml@main
+  build-container-image:
+    name: "Build container image"
+    uses: kubewarden/kubewarden-controller/.github/workflows/container-image.yml@main
+    with:
+      push-image: false
```

Comment:
Can we make `@main` a parameter of the workflow? Adding to previous concerns, on the release workflow we might be releasing a different commit than what `main` points to, and in that timeframe, unit tests, linter... might have changed. Ideally we should be referring to the current commit if possible.

Reply:
Good idea. I think is doable. I'll work on that while checking all the concerns from you and Flavio.